src/components/Awards/Awards.jsx

```diff
@@ -1,45 +1,74 @@
+        </div>
         {/* cards container */}
-        <div className="flex flex-col sm:flex-row justify-center gap-y-8 sm:gap-x-12 p-6">
-
+        <div className="flex flex-col sm:flex-row justify-center gap-y-8 sm:gap-x-20 p-6">
           {/* 1st Runner up */}
-          <div className="w-full sm:w-[30%] bg-white border border-gray-200 rounded-lg shadow-lg transition-transform transform hover:scale-105 hover:shadow-xl flex flex-col justify-center items-center py-10 gap-y-6">
-            <FaMedal className="text-red-500 w-[100px] h-[100px]" /> {/* Medal Icon */}
-            <div className="text-center">
-              <h2 className="text-3xl font-bold text-gray-800">1st Runner up</h2>
-              <h3 className="text-xl font-semibold text-gray-600 mt-2">
-                40,000 LKR
-              </h3>
+          <div className="w-full sm:w-[20%] h-[400px] p-[2px] bg-gradient-to-r from-orange-500 to-orange-950 rounded-lg transition-transform transform flex justify-center items-center order-2 sm:order-none">
+            {/* Inner content with shadow and solid background */}
+            <div className="w-full h-full bg-black shadow-[0_0_15px_10px_rgba(249,115,22,0.7)] rounded-lg flex flex-col justify-center items-center py-10 gap-y-6">
+              <img
+                src="../src/assets/images/awards/award_2.png"
+                alt="Description of image"
+                className="w-[180px] h-[180px] object-contain"
+              />
+              {/* Medal Icon */}
+              <div className="text-center">
+                <h2 className="text-xl font-bold text-white">
+                  1<sup>st</sup> Runner up
+                </h2>
+                <h3 className="text-3xl font-semibold text-white mt-2">
+                  25,000 LKR
+                </h3>
+              </div>
             </div>
           </div>
 
           {/* Champion card */}
-          <div className="w-full sm:w-[30%] bg-white border border-gray-200 rounded-lg shadow-lg transition-transform transform hover:scale-115 hover:shadow-xl flex flex-col justify-center items-center py-10 gap-y-6 scale-110 ">
-
-            <FaTrophy className="text-yellow-500 w-[100px] h-[100px]" /> {/* Trophy Icon */}
-            <div className="text-center">
-              <h2 className="text-3xl font-bold text-gray-800">Champion</h2>
-              <h3 className="text-xl font-semibold text-gray-600 mt-2">
-                30,000 LKR
-              </h3>
+          <div className="w-full sm:w-[20%] h-[400px] p-[2px] bg-gradient-to-r from-orange-500 to-orange-950 rounded-lg transition-transform transform flex justify-center items-center order-1 sm:order-none">
+            {/* Inner content with shadow and solid background */}
+            <div className="w-full h-full bg-black shadow-[0_0_15px_10px_rgba(249,115,22,0.7)] rounded-lg flex flex-col justify-center items-center py-10 gap-y-6">
+              <img
+                src="../src/assets/images/awards/award_1.png"
```

Comment:
please move the image files to the public directory and use directly like this 
`src="/images/awards/award-1.png"`